File: backend/services/file-storage-service/src/main/java/com/lmspilot/filestorage/FileStorageServiceApplication.java
package com.lmspilot.filestorage;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class FileStorageServiceApplication {
    public static void main(String[] args) { SpringApplication.run(FileStorageServiceApplication.class, args); }
}

File: backend/services/file-storage-service/src/main/kotlin/com/lmspilot/filestorage/FileStorageServiceApplication.kt
package com.lmspilot.filestorage

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class FileStorageServiceApplication

fun main(args: Array<String>) {
    runApplication<FileStorageServiceApplication>(*args)
}
